compliance/tags/azure_rg_tags/azure_resource_group_tags.pt

```diff
@@ -0,0 +1,303 @@
+name "Azure: Tag Resources with Resource Group Name"
+rs_pt_ver 20180301
+type "policy"
+short_description "Scan all resources in an Azure Subscription, raise an incident if any resources are not tagged with the name of their Resource Group, and remediate by tagging the resource. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/compliance/tags/azure_rg_tags) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "medium"
+category "Compliance"
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+end
+
+parameter "param_azure_tenant_id" do
+  type "string"
+  label "Azure AD Tenant ID"
+  category "Azure"
+end
+
+parameter "param_azure_sub" do
+  type "string"
+  label "Azure Subscription ID"
+  category "Azure"
+end
+
+parameter "param_tag_key" do
```

Comment:
Is this tag_key parameter always the same? If not, when does it differ. If so, shouldn't it be a constant embedded in code or at least have a default value?
Also FYI parameters do support a "description" field -- The api certainly returns it I think the UI will display it inline rather than only having it embedded in the source.

Reply:
it's going to be customer-specific, so it should be same within a single org